pyproject.toml

```diff
@@ -6,13 +6,10 @@ build-backend = "setuptools.build_meta"
 name = "fake-useragent"
 version = "1.5.1"
 authors = [
-    {name = "Melroy van den Berg", email = "melroy@melroy.org"},
-    {name = "Victor Kovtun", email = "hellysmile@gmail.com"}
-]
-dependencies = [
-    "importlib-resources >= 5.0; python_version < '3.10'",
-    "importlib-metadata ~= 4.0; python_version < '3.8'",
+    { name = "Melroy van den Berg", email = "melroy@melroy.org" },
+    { name = "Victor Kovtun", email = "hellysmile@gmail.com" },
 ]
+dependencies = ["importlib-resources >= 5.0; python_version < '3.10'"]
```

Comment:
Sorry why was here put < 3.10? And not python_version > '3.8'?

Reply:
ah got it. `dependencies = ["importlib-resources >= 5.0; python_version < '3.10'"]` is correct. It means that importlib-resources will be used & downloaded only for python_version below 3.10.